NEWS.md

```diff
@@ -1,6 +1,8 @@
 # gtsummary (development version)
 
-* Updated the default function in add_glance_*(galnce_fun=) for MICE models. (Update glance() method for mice objects in add_glance_table() #1137)
+* No longer removing the strata column from survey objects from the columns that will be summarized in `tbl_svysummary()`. If users previously didn't indicate which variables to summarize with `include=`, then the strata column (if present) will now appear in the summary table. When using a srvyr class object, all columns will be printed in summary table by default. (#1166)
```

Comment:
Here the result will be different for a `survey` or a `srvyr` object. I would say that this is a complexity which is not relevant. The results should not change according to the type of the data set and both should be treated equally. I suspect that we will have misunderstanding if the rule differ for `survey` and `srvyr`.

I would be in favor in having a simple rule: all variables are, by default, returned by `everything()`. If user what a subset, they can use `include=` or `survey::subset()` or `srvyr::select()`.

In such case, we should simple remove `.remove_survey_cols()`